pages/ksphere/kommander/latest/clusters/attach-cluster/index.md

```diff
@@ -8,13 +8,17 @@ excerpt: A guide for attaching an existing Kubernetes cluster using kubeconfig
 
 ## Attach Kubernetes Cluster
 
-Using the **Add Cluster** option, you can attach an already existing cluster, or an existing Konvoy cluster, directly to Kommander. You can access the multi-cluster management and monitoring benefits Kommander provides while keeping your existing cluster on its current provider and infrastructure.
+You can attach an existing cluster directly to Kommander. There are two main ways of doing it, described below, and one requires to create a new service account. If the cluster you wish to attach has been created using Amazon EKS, Azure AKS, or Google GKE, please follow the instructions "Attaching Kubernetes Cluster Using a New Service Accoun".
```

Comment:
> please follow the instructions "Attaching Kubernetes Cluster Using a New Service Account".

is it possible to make this a link pointing to the anchor?

Reply:
I haven't done that as I have moved the instructions "Attaching Kubernetes Cluster Using a New Service Account" right after this text following your other suggestion.